src/components/Vault/VaultStrategy.tsx

```diff
@@ -32,17 +28,6 @@ import ExternalLink from "~/svg/icons/external-link.svg"
 const VaultStrategyButton: FC<VaultProps> = (props) => {
   const [isStrategyOpen, setIsStrategyOpen] = useState(false)
 
-  const { data: vaultTokens, ...vaultTokensQuery } = useVaultTokens(props)
```

Comment:
It's just not necessary to pass these down multiple levels. Updated to fetch the data where it is consumed + added loading states.